pkg/apis/garden/v1beta1/helper/helpers.go

```diff
@@ -713,3 +748,54 @@ func SetZoneForShoot(shoot *gardenv1beta1.Shoot, cloudProvider gardenv1beta1.Clo
 		shoot.Spec.Cloud.Packet.Zones = zones
 	}
 }
+
+// DetermineLatestMachineImageVersion determines the latest MachineImageVersion from a MachineImage
+func DetermineLatestMachineImageVersion(image gardenv1beta1.MachineImage) (*semver.Version, gardenv1beta1.MachineImageVersion, error) {
+	var (
+		latestSemVerVersion       *semver.Version
+		latestMachineImageVersion gardenv1beta1.MachineImageVersion
+	)
+
+	if len(image.Version) > 0 {
```

Comment:
Probably not, butter maybe still better to keep it to be safe?

Reply:
We just discussed: If we want to keep it then you should also check the `Version` field here: https://github.com/gardener/gardener/pull/1177/files/4356fbccb5b442eee2f4d3bcf71dff8dd8b3c004#diff-a3d3f75ad34298fb9696284b0ef772c2R786

If we don't want to keep it because we want to rely on the conversions (which probably is perfectly fine) then let's remove all usage of the `Version` field when working with the `v1beta1` version.
Thanks @timuthy.

Reply:
No. thanks !